keyboards/dz60/keymaps/mpaarating/readme.md

```diff
@@ -0,0 +1,21 @@
+# DZ60
```

Comment:
General comment:

The template you started from to write this file is really intended to be used for keyboard readmes. For a keymap readme a lot of this file is redundant. Lines 5, 7, 9-10, and 21 don't really serve any purpose here. This file should detail specifics about your keymap, rather than the hardware (unless you've modded the hardware somehow, like adding RGB underglow to a PCB that doesn't include it or the supporting pads for the LEDs).

```suggestion
# mpaarating's DZ60 keymap
```

Reply:
That makes sense. I had wondered about that but left it in to favor more information than less. I'll pull it out!